packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -168,6 +170,12 @@ async function createWalletAddress(
   }
 
   try {
+    const tenantId = options.tenantId
+      ? options.tenantId
+      : deps.config.operatorTenantId
+    const asset = await deps.assetService.get(options.assetId, tenantId)
```

Comment:
```suggestion
    const asset = await deps.assetService.get(options.assetId, options.tenantId)
```
I think this should be sufficient?

Reply:
I am blind as a bat. I thought I removed that already! 🥹